nifi-nar-bundles/nifi-gcp-bundle/nifi-gcp-processors/pom.xml

```diff
@@ -80,6 +80,12 @@
             <artifactId>json</artifactId>
             <version>1.8</version>
         </dependency>
+        <dependency>
+            <groupId>org.apache.nifi</groupId>
+            <artifactId>nifi-json-utils</artifactId>
+            <version>1.7.0-SNAPSHOT</version>
```

Comment:
This causes issue when starting NiFi. Reason seems to be `google-auth-library-oauth2-http` comes with `jackson-core:2.1.3` as its dependency. We can't exclude it because of the version mismatch. Even the latest version of `google-auth-library-oauth2-http` uses `jackson-core:2.1.3`